Utils/Campaign builder/Unit1.pas

```diff
@@ -218,11 +245,15 @@ procedure TForm1.btnSaveCMPClick(Sender: TObject);
   end;
 
   dlgSaveCampaign.InitialDir := ExtractFilePath(dlgOpenCampaign.FileName);
+
+  dlgSaveCampaign.FileName := 'info';
+
   if not dlgSaveCampaign.Execute then Exit;
 
   C.SaveToFile(dlgSaveCampaign.FileName);
-
   fSprites.SaveToRXXFile(ExtractFilePath(dlgSaveCampaign.FileName) + 'images.rxx');
+  if chCreateLibxTemplate.Checked then

```

Comment:
Do we need a checkbox for this? If the eng.libx file doesn't exist you probably always want to create it.